openslides/motions/static/templates/motions/motion-detail.html

```diff
@@ -273,17 +273,42 @@ <h3 translate>Version history</h3>
         </ul>
       </div>
 
-      <!-- log -->
-      <button type="button" class="btn btn-default spacer" ng-click="isCollapsed = !isCollapsed" translate>
-        Show history
-      </button>
-      <div uib-collapse="isCollapsed">
+      <div class="row spacer">
+        <div class="col-md-1">
+
+          <!-- switching line numbering mode -->
+          <div class="dropdown">
```

Comment:
Please move this drop down field above the motion text (below text and reason it is very hard to switch line numbering mode).

I prefere a  [button group](http://getbootstrap.com/components/#btn-groups-single) - not a drop down list - for this switch element.